pandas/core/common.py

```diff
@@ -2672,6 +2672,9 @@ def is_list_like(arg):
      return (hasattr(arg, '__iter__') and
             not isinstance(arg, compat.string_and_binary_types))
 
+def is_named_tuple(arg):
+    return isinstance(arg, tuple) and hasattr(arg, '_fields')
```

Comment:
cool

we might have named tuples elsewhere that could use he is_ function can u give a check